Automattic/wp-calypso
# Plugin Management: Update button style and implement show plugin update status in Jetpack Cloud

#### Proposed Changes

This PR changes the "Update plugin" button style in Jetpack Cloud and implements showing plugin update status.

**Screenshots**

**Multi-sites view - large screen**

<img width="1447" alt="Screenshot 2022-09-06 at 8 02 23 PM" src="https://user-images.githubusercontent.com/10586875/188667939-ae72362f-3ab7-4dfa-ac22-3d1421da74c0.png">
<img width="1527" alt="Screenshot 2022-09-06 at 8 02 29 PM" src="https://user-images.githubusercontent.com/10586875/188667968-875ec965-e979-4bef-8ea4-8f6c89b910c2.png">
<img width="1482" alt="Screenshot 2022-09-06 at 8 02 32 PM" src="https://user-images.githubusercontent.com/10586875/188667979-292892d9-f7c4-4ba5-bc0b-38b32e939a3c.png">

**Multi-sites view - small screen**

<img width="368" alt="Screenshot 2022-09-06 at 8 00 53 PM" src="https://user-images.githubusercontent.com/10586875/188668085-eca1b399-d08e-46bf-a662-55c2487ed8e0.png">
<img width="374" alt="Screenshot 2022-09-06 at 8 01 07 PM" src="https://user-images.githubusercontent.com/10586875/188668106-6d304f6d-95d1-4997-8ffe-c12b8d6fb90e.png">

**Single-site view - large screen**

<img width="1401" alt="Screenshot 2022-09-06 at 8 03 00 PM" src="https://user-images.githubusercontent.com/10586875/188668239-bc163a76-cba5-444b-a43f-cb2f703e5725.png">

Single-site view - small screen

<img width="370" alt="Screenshot 2022-09-06 at 8 03 19 PM" src="https://user-images.githubusercontent.com/10586875/188668549-d2223cb0-9f94-455f-9a61-fbf136e749b9.png">
<img width="368" alt="Screenshot 2022-09-06 at 8 03 43 PM" src="https://user-images.githubusercontent.com/10586875/188668582-5eb7f6aa-041e-4c4f-bb0b-b0851e7e4dc1.png">
<img width="365" alt="Screenshot 2022-09-06 at 8 03 54 PM" src="https://user-images.githubusercontent.com/10586875/188668595-d6918bf1-b3e6-4b0f-b4bd-e2c42b978450.png">
<img width="361" alt="Screenshot 2022-09-06 at 8 15 51 PM" src="https://user-images.githubusercontent.com/10586875/188668601-2c9bed11-214e-455d-81ac-ae72522043a5.png">
<img width="367" alt="Screenshot 2022-09-06 at 8 16 01 PM" src="https://user-images.githubusercontent.com/10586875/188668608-1eac01e1-973f-49b3-ac1f-ff3113eb3785.png">

#### Testing Instructions

**Prerequisites**
- Since this change is made specifically for agencies, you must set yourself (partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type when you're done with testing.
- Please ensure you have an outdated plugin on some of your sites. If you want to know how to find an older version of a specific plugin, read [this](https://wordpress.org/support/topic/old-version-of-plugin-2/).

**Instructions**
- Checkout this branch locally
- Run `yarn start-jetpack-cloud`
- Visit http://jetpack.cloud.localhost:3000/plugins/manage
- Verify that the update button is displayed correctly (as per design) for the plugin requiring an update
- Choose a specific site from the sidebar (i.e. `Switch site`) and then click "Plugins" from the sidebar
- Verify that the update button is displayed correctly (as per design) for the plugin requiring an update
- Click "Edit all", choose some plugins and then click the "Update" button
- Verify that the inline feedback notice (the status) is displayed correctly and makes sense

**Test update retry**
- Create a new JN site, and add the following in `wp-config.php`, to make the "Update plugin" request always fail.
```
define( 'DISALLOW_FILE_MODS', true );
```
- Verify that "Retry" button is working as expected

#### Pre-merge Checklist






Related to 1202518759611394-as-1202774994084578 & 1202518759611394-as-1202927891445246
